tests/v1.0/fail/info-missing-title.yaml

```diff
@@ -0,0 +1,4 @@
+overlay: 1.0.0
+info:
+  version: 1.0.0
+actions: []
```

Comment:
Better add a valid action here to make sure that the `overlay` field causes the fail